### ISSUE TITLE
UnlockAllOrientations not working for the first call

### ISSUE THREAD

**Issue by abdrmdn** [wonday/react-native-orientation-locker#66]
Having this code in the screen i want to use
```
   componentDidMount() {
        Orientation.unlockAllOrientations();
    }

    componentWillUnmount() {
        Orientation.lockToPortrait();
    }
```

will work only the second time i go to that screen, then i figured it could be the issue that its not set right in the beginning.
workaround was : 
```
   componentDidMount() {
        Orientation.lockToPortrait(); // <----- This will reset stuff before the first unlock
        Orientation.unlockAllOrientations();
    }

    componentWillUnmount() {
        Orientation.lockToPortrait();
    }
```

But, it feels wrong that i have to lock and then unlock, i think this is a bug that needs to be fixed :)

**Comment by wonday** [comment]
I have fixed it, You can try v1.0.16.

**Comment by abdrmdn** [comment]
Great thanks a bunch.
